src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -46,10 +54,33 @@ export class PaymentRoute extends Component<PaymentProps, PaymentState> {
       country: "US",
     },
     hideBillingAddress: true,
+    errorMessage: null,
+    isComittingMutation: false,
+  }
+
+  onContinueButtonPressed = () => {
+    const billingAddress = this.getSelectedBillingAddress()
+
+    this.setState({ isComittingMutation: true }, () => {
+      this.props.stripe.createToken(billingAddress).then(({ error, token }) => {
+        console.log("Received Stripe token:", error, token)
```

Comment:
Maybe we can move a `console.error` into the error block and otherwise not log out